abb_driver/rapid/ROS_motion.mod

```diff
@@ -54,6 +54,7 @@ PROC main()
         IF (trajectory_size > 0) THEN
             FOR current_index FROM 1 TO trajectory_size DO
                 target.robax := trajectory{current_index}.joint_pos;
+                target.extax := trajectory{current_index}.extax_pos;
 
                 skip_move := (current_index = 1) AND is_near(target.robax, 0.1);

```

Comment:
Would this also not need to check the pose(s) of the external axes?

Reply:
Fixed it on https://github.com/ros-industrial/abb/pull/150/commits/3b1d413b265378102ed51f17b4a90acd19b9b4b8